source/brailleDisplayDrivers/hims.py

```diff
@@ -116,6 +116,25 @@ def _get_keys(self):
 		})
 		return keys
 
+
+class BrailleEdge2S(BrailleEdge):
+	"""This device is the BrailleEdge which doesn't use the hims driver.
+	It only uses a SPP connection.
+	"""
+	usbId = "VID_1A86&PID_55D3"
+
+	def _get_keys(self) -> dict[str, str]:

```

Comment:
If the keys on the new Edge are exactly the same as the keys on the old device, I think you can remove this override.

Reply:
I think my post lacked explanation. Same device means focus to users. In the device, key packet data little bit changed.